app/ui/browser-modern/model/page-state.js

```diff
@@ -14,6 +14,10 @@ import Immutable from 'immutable';
 
 const PageState = Immutable.Record({
   load: undefined,
+  code: undefined,
+  description: undefined,
+  certificate: undefined,
+  url: undefined,
```

Comment:
Why is there an `url` property in the page state? There's a `location` in the page model. Having both is confusing.